freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/model/repair/RepairInstancesRequest.java

```diff
@@ -25,6 +25,9 @@
     @ApiModelProperty(ModelDescriptions.INSTANCE_ID)
     private List<String> instanceIds;
 
+    @ApiModelProperty(ModelDescriptions.FREEIPA_INSTANCE_COUNT_BY_GROUP)
+    private Integer instanceCountByGroup;
```

Comment:
it's a bit odd and confusing for me. So with repair endpoint/request will we be able to scale the size of the FreeIPA cluster? Is it really a repair functionality? also the naming, `instanceCountByGroup` a bit odd, as we don't really specify any group in the request